src/components/Dialog/index.tsx

```diff
@@ -0,0 +1,58 @@
+import MuiDialog, { DialogProps as MuiDialogProps } from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import CloseIcon from '@mui/icons-material/Close';
+import {Button} from "../Button";
+import React from "react";
+export interface DialogTitleProps {
+    id: string;

```

Comment:
let's remove 'id' since there is no need to pass it to <HistorySelectionDialogTitle> and moreover there is no need to use id, while it can be a className with BEM